honeybee_energy/programtype.py

```diff
@@ -63,5 +82,567 @@ def name(self):
+                "space_type": "MediumOffice - OpenOffice",
+                "lighting_standard": "ASHRAE 90.1-2013",
+                "lighting_per_area": 0.98,
+                "lighting_per_person": None,
+                "additional_lighting_per_area": None,
+                "lighting_fraction_to_return_air": 0.0,
+                "lighting_fraction_radiant": 0.7,
+                "lighting_fraction_visible": 0.2,
+                "lighting_schedule": "OfficeMedium BLDG_LIGHT_SCH_2013",
+                "ventilation_standard": "ASHRAE 62.1-2007",
+                "ventilation_primary_space_type": "Office Buildings",
+                "ventilation_secondary_space_type": "Office space",
+                "ventilation_per_area": 0.06,
+                "ventilation_per_person": 5.0,
+                "ventilation_air_changes": None,
+                "minimum_total_air_changes": None,
+                "occupancy_per_area": 5.25,
+                "occupancy_schedule": "OfficeMedium BLDG_OCC_SCH",
+                "occupancy_activity_schedule": "OfficeMedium ACTIVITY_SCH",
+                "infiltration_per_exterior_area": 0.0446,
+                "infiltration_schedule": "OfficeMedium INFIL_SCH_PNNL",
+                "gas_equipment_per_area": None,
+                "gas_equipment_fraction_latent": None,
+                "gas_equipment_fraction_radiant": None,
+                "gas_equipment_fraction_lost": None,
+                "gas_equipment_schedule": None,
+                "electric_equipment_per_area": 0.96,
+                "electric_equipment_fraction_latent": 0.0,
+                "electric_equipment_fraction_radiant": 0.5,
+                "electric_equipment_fraction_lost": 0.0,
+                "electric_equipment_schedule": "OfficeMedium BLDG_EQUIP_SCH_2013",
+                "heating_setpoint_schedule": "OfficeMedium HTGSETP_SCH_YES_OPTIMUM",
+                "cooling_setpoint_schedule": "OfficeMedium CLGSETP_SCH_YES_OPTIMUM"
+                }
+
+            data_schedules: Dictionary representation of all schedules in the
+                OpenStudio standards gem. Each schedule should have the scheule
+                name as a key and the value should be a list of individual
+                ScheduleDay dictionary descriptions from the standards gem that
+                together define a complete ScheduleRuleset.
+        """
+        pr_type_name = data['space_type']
+        people = None
+        lighting = None
+        electric_equipment = None
+        gas_equipment = None
+        infiltration = None
+        ventilation = None
+        setpoint = None
+
+        if data['occupancy_schedule'] is not None:
+            occ_sched = ScheduleRuleset.from_standards_dict(
+                data_schedules[data['occupancy_schedule']])
+            act_sched = ScheduleRuleset.from_standards_dict(
+                data_schedules[data['occupancy_activity_schedule']])
+            occ_density = data['occupancy_per_area'] / 92.903
```

Comment:
Is there a plan for OpenStudio standards to have a SI version? I'm asking since it might make sense to add an input for units to be set to SI and / or IP or we can probably do that when the SI version will be available if ever!

Reply:
That would be nice. The sense I had gotten from Dan a while back is that it's not really in the plans right now (or it's not in the plans for as long as the spreadsheet lives since it seems the people adding/maintaining the US standards are IP people).  Matt and Marely will know better and I will ask when I get the chance to see them.  I'll add it if they think it's in the works.

BTW, I am thinking that all of these `from_standards_dict` methods will get pulled out of the `honeybee_energy` library and put into the `honeybee_standards` library once we set it up. These methods aren't really practical unless you have the standards gem data present so I feel like it makes sense for `honeybee_standards` to extend the `honeybee_energy` objects with these methods.